README.md

```diff
@@ -113,3 +113,29 @@ We are passionate about improving this project, and glad to accept help to make
 
 **Feature Requests:** If you have a need that is not met by the current implementation, please let us know (via a new issue).
 This feedback is crucial for us to deliver a useful product. Do not assume that we have already thought of everything, because we assure you that is not the case.
+
+## Naming convention
+
+By adding support for a new resource, 3 files are required: self-contained module, test and example. Below their naming conventions.
```

Comment:
I suggest **The following is a summary of the code structure and naming conventions for the oneview-ansible modules.** instead of **Below their naming conventions.**